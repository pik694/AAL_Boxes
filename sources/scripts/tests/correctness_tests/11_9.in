11
20 4 85
93 62 1
57 99 16
10 66 3
31 65 80
58 45 13
55 94 12
79 69 18
1 99 7
98 44 84
64 7 62

