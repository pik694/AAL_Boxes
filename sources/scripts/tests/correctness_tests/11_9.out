8 3 0 6 2 1 10 7 5 4 9 
850799->557073
